### DISCUSSION TITLE
No reflections in QuickLook view

### DISCUSSION THREAD

**Discussion by DanieleMagri** [google/model-viewer#3535] [Q&A]
When I use webXR for the AR, the model is rendered exactly as it is rendered in the model viewer (it probably uses the same three.js scene for the AR view, right?)
But when I use QuickLook with an iPhone SE 2nd gen, there are no reflections, neither in AR or Object mode.

**Comment by elalish** [comment]
Are you creating your own USDZ or using auto-generation? Some screenshots would help too. FYI @grorg (Apple).

**Reply by DanieleMagri** [reply]
I'm using the built in auto-generation, as soon as I can I'll send some screenshots
